src/test/scala/nl/knaw/dans/easy/fedoratobag/DdmSpec.scala

```diff
@@ -1044,7 +1044,37 @@ class DdmSpec extends TestSupportFixture with EmdSupport with AudienceSupport wi
     }
   }
 
-  it should "render an invalid number of dates created" in {
+  it should "parse dates into reasonable dates" in {
+    val emd = parseEmdContent(Seq(
+      emdTitle, emdCreator, emdDescription,
+          <emd:date>
+            <eas:created eas:scheme="W3CDTF" eas:format="MILLISECOND">20180223-01-01T00:10:34.000+01:00</eas:created>
+            <eas:available eas:scheme="W3CDTF" eas:format="DAY">2017-09-30T17:47:36.978+02:00</eas:available>
```

Comment:
What about
```suggestion
            <eas:available eas:scheme="W3CDTF" eas:format="DAY">20170930-01-01T17:47:36.978+02:00</eas:available>
```

Reply:
the thing is that only when eas:format=MILLISECOND do these strange dates appear. The MILLISECOND was the last-resort for the poor-mans-parser in the EMD library. So 
```
<eas:available eas:scheme="W3CDTF" eas:format="DAY">20170930T17:47:36.978+02:00</eas:available>
```
is not valid EMD, but with format=MILLISECOND it is ...